cli/ceph/orch/orch.py

```diff
@@ -99,3 +99,16 @@ def ps(self, **kw):
         if isinstance(out, tuple):
             return out[0].strip()
         return out
+
+    def label_rm(self, hostname, labelname):
```

Comment:
this function should be moved here - https://github.com/red-hat-storage/cephci/blob/master/cli/ceph/orch/label.py

Reply:
Thanks @vamahaja
Updated